modules/apps/site/site-cms-site-initializer/test/js/main/spaces/NewSpace.test.tsx

```diff
@@ -89,12 +72,17 @@ describe('NewSpace', () => {
 		await userEvent.type(descriptionInput, spaceDescription);
 		expect(descriptionInput).toHaveValue(spaceDescription);
```

Comment:
I would remove this expect, is not necessary, as well as the above `expect(spaceNameInput).toHaveValue(spaceName);`

We are kind of checking this values in the last expect with `toHaveBeenCalledWith`